rcl_interfaces/msg/RclHeader.msg

```diff
@@ -0,0 +1,5 @@
+builtin_interfaces/Time stamp
+# A fully qualified ROS node name
+string sender_node_name
+
+# TODO(mikaelarguedas) consider a receiver node name as well
```

Comment:
The reciever/requester does not change the interpretation of the data thus I would advocate not to include that.

Reply:
Yeah, The use case I was trying to cover here (but am focused on parameter events right now so there may be others) is that it would be pretty convenient to know what node triggered a parameter change. But that seems to be very parameter event specific and may not belong in a more generic rcl header used by other messages.
Though it could be added in the ParameterEvent and ParameterEventDescriptor messages.

Reply:
Yeah, I think that sort of information would make sense to embed in those two messages directly not in the generic header.